src/styles/main.scss

```diff
@@ -1 +1,79 @@
-// put code here
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+}
+
+
+.watch {
+  box-sizing: border-box;
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 3px dashed black;
+  position: relative;
+
+  &__center {
+    width: 30px;
+    height: 30px;
+    background-color: orange;
+    position: absolute;
+    border-radius: 50%;
+    inset: 0;
+    margin: auto;
+  }
+
+  &__hour-hand {
+    width: 9px;
+    height: 100%;
+    background-color: blue;
+    position: absolute;
+    left: 0;
+    right: 0;
+    margin: auto;
+    animation: rotate-hour-hand 55s linear infinite;
+  }
+
+  &__hour-hand::before {
+    position: absolute;
+    content: "";
+    background: blue;
+    height: 21vmin;
+    width: 9px;
+  }
+
+  @keyframes rotate-hour-hand {
+    to {
+      transform: rotate(360deg);
+    }
+  }
+
+  &__minute-hand {
+    width: 3px;
+    height: 100%;
+    background-color: green;
+    position: absolute;
+    left: 0;
+    right: 0;
+    margin: auto;
+    animation: rotate-minute-hand 5s linear infinite;
+  }
+
+  &__minute-hand::before {
+    content: '';
+    position: absolute;
+    width: 3px;
+    height: 41vmin;
+    background-color: green;
+  }
+
+  @keyframes rotate-minute-hand {
+    to {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
both minute hand and hour one have the same animation. Reuse
```suggestion
```

Reply:
still not resolved
![image](https://github.com/mate-academy/layout_stop-watch/assets/55200530/14c2e5d6-acf9-4c96-ae82-5f896109ee96)